Formula/httest.rb

```diff
@@ -18,15 +18,15 @@ class Httest < Formula
   depends_on "lua"
 
   def install
+    (buildpath/"brew_include").install_symlink Formula["pcre"].opt_include => "pcre"
+    ENV.prepend "CPPFLAGS", "-I#{buildpath}/brew_include"
```

Comment:
Why do we need to create a symlink, instead of `ENV.prepend "CPPFLAGS", "-I#{Formula["pcre"].opt_include}"`?
And why is libpcre not linked?